docs/pages/guides/local-first.mdx

```diff
@@ -38,6 +38,17 @@ A comprehensive list of tools is available on the ["Local-first software" commun
 
 One way to think of local-first tools is to group them by the following categories: persistence, state management, and syncing. Some tools will fit into multiple categories if they handle multiple aspects of the problem. Syncing can be further subdivided into syncable data structures and transport layers.
 
+### Legend-State
```

Comment:
To solve the warning here, we'll add this to the exception in our vale rules. I'll follow up with a fix for this in a separate PR.